pallets/automation-time/src/tests.rs

```diff
@@ -309,12 +391,78 @@ fn cancel_works_for_scheduled() {
 	})
 }
 
+#[test]
+fn cancel_works_for_multiple_executions_scheduled() {
```

Comment:
is there a test that checks for cancel when one of the task has already been executed?